swift-sdk/Internal/InAppManager.swift

```diff
@@ -35,7 +35,7 @@ protocol IterableInternalInAppManagerProtocol: IterableInAppManagerProtocol, InA
     func remove(message: IterableInAppMessage, location: InAppLocation, source: InAppDeleteSource, inboxSessionId: String?)
 }
 
-class InAppManager: NSObject, IterableInternalInAppManagerProtocol {
+class InAppManager: NSObject {
```

Comment:
Nit: For discussion: In this case the entire purpose of InAppManager is that it is an implementation of IterableInAppManagerProtocol. I think it may be better to not move the protocol work out of this.

Usually when we have `extension ClassX: protocol PX ` we are implementing a secondary protocol implementation other than the primary purpose of the class.

For example, AppDelegate can be implementing the `UNUserNotificationCenterDelegate`. In that case we will have 
`extension AppDelegate: UNUserNotificationCenterDelegate`

Reply:
Maybe we could put a `// MARK: IterableManagerProtocol implementation.` and put all methods there. 

Also if a protocol has variables you will not be able to put the the variable declaration into the `extension ClassX: protocol PX` section.